aten/src/ATen/TensorNames.h

```diff
@@ -29,19 +29,25 @@ namespace at { namespace namedinference {
 // to A, `tensor` would have duplicate names [A, A]. Therefore we need to check
 // tensor.names [A, None] for the existence of A.
 struct CAFFE2_API TensorName {
+  TensorName()
```

Comment:
where is this constructor needed?

Reply:
Here: https://github.com/pytorch/pytorch/pull/29307/files#diff-adc780bb266d74ff645db995562a459cR76

Reply:
`std::vector<TensorName>::resize` resizes the container. If it is resized to a larger size, then the container needs to insert new elements. The value of the new elements is either passed in via `resize`, or they are default constructed (the TensorName constructor gets called).

Now that I think about it more, there is a good way around this problem that I will go and implement to remove the `initialized_` flag. Instead of resizing the container, I can prepend to it names from the other list of names.

Reply:
I've removed the `initialized_` flag